examples/hello_world/make/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2020, Synopsys, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-3-Clause license found in
+# the LICENSE file in the root directory of this source tree.
+#
+
+TARGET ?= NATIVE
+
+# From rules.mk
+BACKSLASH=\$(nullstring)
+ifneq ($(ComSpec)$(COMSPEC),)
+	O_SYS=Windows
+	RM=del /F /Q
+	RMDIR=rmdir /Q /S
+	MKDIR=mkdir
+	PS=$(BACKSLASH)
+else
+	O_SYS=Linux
+	RM=rm -rf
+	RMDIR=rm -rf
+	MKDIR=mkdir -p
+	PS=/
+endif
+
+# Set target specific setting for NATIVE build
+ifeq ($(TARGET),NATIVE)
+BUILD_FOLDER = ..$(PS)..$(PS)..$(PS)outputs_x86
+ifeq ($(UNAME),Windows)
+TOOL_CHAIN_OPTIONS = -AWin32
+else
+TOOL_CHAIN_OPTIONS =
+endif
+
+else
+# Set target specific setting for ARC build
+ifeq ($(TARGET),ARC)
+BUILD_FOLDER = ../../../outputs_mwdt
```

Comment:
There is a chance that result project can use ARC GNU as toolchain. In this context I think it's confusing to stick to *_mwdt postfix